background/constants/networks.ts

```diff
@@ -494,19 +297,203 @@ export const DEFAULT_QUAI_LOCAL = {
   ]
 } as Network
 
-export const DEFAULT_QUAI_NETWORKS = [
-  DEFAULT_QUAI_TESNTET,
-  DEFAULT_QUAI_DEVNET,
-  DEFAULT_QUAI_LOCAL
+export const QUAI_NETWORK: EVMNetwork = {
+  name: "Quai Network", // maybe add "Colosseum"
+  baseAsset: QUAI,
+  chainID: "9000",
+  family: "EVM",
+  chains: DEFAULT_QUAI_TESTNET.chains,
+  isQuai: true,
+}
+
+export const QUAI_NETWORK_LOCAL: EVMNetwork = {
+  name: "Quai Network Local",
+  baseAsset: QUAI_LOCAL,
+  chainID: "1337",
+  family: "EVM",
+  chains: DEFAULT_QUAI_LOCAL.chains,
+  isQuai: true,
+}
+
+
+export const DEFAULT_NETWORKS = [
+  QUAI_NETWORK,
+  QUAI_NETWORK_LOCAL,
 ]
 
-export class QuaiContext {
+export function isBuiltInNetwork(network: EVMNetwork): boolean {
+  return DEFAULT_NETWORKS.some(
+    (builtInNetwork) => builtInNetwork.chainID === network.chainID
+  )
+}
+
+export const DEFAULT_NETWORKS_BY_CHAIN_ID = new Set(
+  DEFAULT_NETWORKS.map((network) => network.chainID)
+)
+
+export const FORK: EVMNetwork = {
+  name: "Ethereum",
+  baseAsset: ETH,
+  chainID: process.env.MAINNET_FORK_CHAIN_ID ?? "69",
```

Comment:
If placeholder we should indicate as such, maybe with a comment.

Reply:
Added a comment